resources/lib/common/logging.py

```diff
@@ -8,15 +8,21 @@
     See LICENSES/MIT.md for more information.
 """
 from __future__ import absolute_import, division, unicode_literals
+
 from functools import wraps
-from time import clock
 
 from future.utils import iteritems
-
 import xbmc
-
 from resources.lib.globals import g
 
+try:
+    from time import perf_counter
+
+    def clock():
+        return perf_counter() * 1.0e-6
+except ImportError:
+    from time import clock
+
```

Comment:
This is not good, avoid put functions on the header file and where possible avoid exceptions

This can be implemented as `perf_clock` function inside logging.py and use the `hasattr` to select clock or perf_counter method to avoid exception

Reply:
and remove the moltiplication `* 1.0e-6`

Reply:
to add also a comment inside function to explain in what py version clock is removed

Reply:
@CastagnaIT According to https://docs.python.org/3/howto/pyporting.html , using exceptions is an idiomatic way. The rest is re-worked.